libraries/AP_NavEKF2/AP_NavEKF2_core.h

```diff
@@ -449,6 +449,7 @@ class NavEKF2_core : public NavEKF_core_common
 
     struct baro_elements : EKF_obs_element_t {
         ftype       hgt;
+        bool corrected;
```

Comment:
I don't think we need to update NavEKF2 for this.

We don't touch EKF2 much, just to keep it as a very stable thing.